help_function/src/onedpl_test_group_load.cpp

```diff
@@ -0,0 +1,106 @@
+// ====------ onedpl_test_group_load.cpp---------- -*- C++ -* ----===////
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//
+// ===----------------------------------------------------------------------===//
+#include <sycl/sycl.hpp>
+#include <dpct/dpct.hpp>
+#include <dpct/dpl_utils.hpp>
+#include <iostream>
+
+
+bool helper_function(const int* ptr,const char* func_name){

```

Comment:
`helper_function` should probably be renamed to describe what it is doing.

Reply:
Yes added comment